src/java/org/apache/cassandra/db/Directories.java

```diff
@@ -1197,6 +1197,11 @@ public static boolean isSecondaryIndexFolder(File dir)
         return dir.name().startsWith(SECONDARY_INDEX_NAME_SEPARATOR);
     }
 
+    public static boolean isSecondaryIndexFolder(Path dir)
+    {
+        return dir.getFileName().toString().startsWith(SECONDARY_INDEX_NAME_SEPARATOR);
```

Comment:
`getFileName().toString()` really always hated that pattern =/

Reply:
`import static org.apache.cassandra.io.util.PathUtils.filename`

`filename(dir)`